### DISCUSSION TITLE
looking for migration guide to PF 14

### DISCUSSION THREAD

**Discussion by hgfrey** [primefaces/community#2138] [PrimeFaces] [answered]
Where can I find the migration guide for version 13 to 14 or 12 to 14? This one existed in March: "https://primefaces.github.io/primefaces/12_0_0/#/.../migrationguide/14_0_0?id=datatable-selection", but it is no longer available.

Many thanks for the help

**Comment by melloware** [comment, marked as answer]
Here are all the migration guides: https://primefaces.github.io/primefaces/15_0_0/#/../migrationguide/migrationguide
